src/multiphonon/ui/getdos_deprecated.py

```diff
@@ -1,4 +1,8 @@
 # -*- python -*-
+# This algorithm should be deprecated with the deletion of ipywe. This algorithm is not covered in tests.
```

Comment:
Can you include in your PR summary how to import this to have it as a reference, please?

Reply:
Yes, there is an added instructions now